docs/src/modules/components/overview/mainDemo/DigitalClock.tsx

```diff
@@ -14,6 +14,7 @@ import {
 } from '@mui/x-date-pickers/PickersLayout';
 import { renderMultiSectionDigitalClockTimeView } from '@mui/x-date-pickers/timeViewRenderers';
 import { TimeView } from '@mui/x-date-pickers/models';
+// eslint-disable-next-line no-restricted-imports
 import { TimePickerViewRenderers } from '@mui/x-date-pickers/TimePicker/shared';
```

Comment:
This was clearly left by mistake (ESLint not working correctly 🙈 )
Could we re-export the type from the index and update the import to `@mui/x-date-pickers/TimePicker`?